api/src/main/java/org/apache/iceberg/actions/RewriteDataFiles.java

```diff
@@ -171,6 +171,17 @@ default RewriteDataFiles zOrder(String... columns) {
    */
   RewriteDataFiles filter(Expression expression);
 
+  /**
+   * A user provided branch which the rewrite happens on the target branch can also be controlled
+   * via WAP(write-audit-publish)
+   *
+   * @param branch the branch where the rewrite happens
+   * @return this for chaining
```

Comment:
to match the others: `this for method chaining`?

Reply:
i also  think we should leave it to match all other returns